### PR TITLE
Complete Random Menu Generator Project

### PR DESCRIPTION
**Pull request by acmei**
I completed Lunch and half of Dinner. The program will accept input from the user, asking how many menu items they would like to generate.

### CODE REVIEW

File: menu.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+puts "How many menu items would you like to create?"
+menu_items = gets.chomp.to_i
+
+# Create 3 arrays, 1. food adjectives, 2. food preparation 3. foods (aka rare animals)
+adj 		= ["Melty", "Spicy", "Buttered", "Gourmet", "Fresh", "Hot", "Crunchy", "Sweet", "Savory", 
+			"Marinated", "Chocolatey", "Flaming", "Creamy", "Polka-dotted", "Chunky"]
+prep 		= ["pan-fried", "sauteed", "steamed", "roasted", "baked", "boiled", "broiled", "fried", 
+			"mashed", "breaded", "scorched", "barbecue", "slow-roasted", "braised", "chopped", "raw", 
+			"sous vide", "poached"]
+rare_animals 	= ["panda", "unicorn", "manatee", "wooly mammoth", "bald eagle", "gorilla", "emperor penguin",
```
Comment (kariabancroft) [inline menu.rb]: Moderately disturbed by your rare_animals collection - but the "crazy" factor definitely holds true :+1:

File: menu.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# How many menu items would the user like to see?
+puts "Welcome to the crazy random menu generator!"
+puts "How many menu items would you like to create?"
+menu_items = gets.chomp.to_i
```
Comment (kariabancroft) [inline menu.rb]: Things to think about for the future: What will happen if the user does not enter an integer? What happens if the user enters an integer whose value is larger than the # of items you have in your lists?

File: menu.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+			"sous vide", "poached"]
+rare_animals 	= ["panda", "unicorn", "manatee", "wooly mammoth", "bald eagle", "gorilla", "emperor penguin", 
+			"blue whale", "hammerhead shark", "toucan", "chimpanzee", "hummingbird", "sea turtle", "rhino", 
+			"monarch butterfly", "sloth"]
+
+# Print menu items in an ordered list based on requested number of items by user
+count = 0
+while count < menu_items
+	count += 1
+	puts "#{count}. #{adj.sample} #{prep.sample} #{rare_animals.sample}"
```
Comment (kariabancroft) [inline menu.rb:27]: Great use of string interpolation here - nice and easy to read what this will be outputting

File: menu.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+end
```
Comment (kariabancroft) [inline menu.rb]: It is generally a good idea to have a newline character at the end of your file. I know that there is a plugin in atom to do this - I'm guessing there is if you're using sublime text too.

**Comment by kariabancroft** [comment]
Great job! Your use of the iterator and string interpolation makes your code clear and concise. Future error checking will be necessary in more advanced projects to be sure those pesky users can't break your code :+1:

File: menu.rb (hunk 1 above)
Comment (acmei) [inline menu.rb]: Thank you.

**Pull request merged**